PluralKit.Bot/Commands/MemberCommands.cs

```diff
@@ -58,6 +58,21 @@ public MemberCommands(IDataStore data, EmbedService embeds, ProxyCacheService pr
             await _proxyCache.InvalidateResultsForSystem(ctx.System);
         }
         
+        public async Task MemberRandom(Context ctx)
+        {
+            ctx.CheckSystem();
+
+            var randGen = new System.Random(); 
+            //Maybe move this somewhere else in the file structure since it doesn't need to get created at every command
+
+            var members = await _data.GetSystemMembers(ctx.System);
+            if (members == null || !members.Any())
+                throw Errors.NoMembersError;
+            var randMember = members.OrderBy(x=> randGen.Next()).First();
```

Comment:
Since `members` is now a list (after calling `.ToList()`), you can just get a random index and pick that one instead of sorting the entire enumerable.